pages/pil/unscoped/courses/schema/index.js

```diff
@@ -0,0 +1,41 @@
+const { toArray } = require('../../../../../lib/utils');
```

Comment:
When the require path is longer than the code that it's requiring it's ok to `Lodash.castArray(val || [])`

Reply:
it does more than that I think

Reply:
also I do plan to fix these paths soon

Reply:
I looked, it does not.